src/code/irqmgr.c

```diff
@@ -2,8 +2,10 @@
  * @file irqmgr.c
  *
  * This file implements a manager for forwarding three key system interrupt events to
- * registered clients. The architecture of this system appears to be derived in part from
- * the libultra sched module.
+ * registered clients.
+ * Together with sched.c, these systems implement the libultra video and task scheduling
```

Comment:
"libultra video and task scheduling"? what is "video" refering to?

Reply:
The "video" part is meant to refer to managing framebuffer swaps (sched) and forwarding retrace events to clients (irqmgr)